src/stdlib.c

```diff
@@ -10,23 +10,40 @@ void* malloc(size_t size) {
   return pvPortMalloc(size);
 }
 
+void* __wrap_malloc(size_t size) {
+  return malloc(size);
+}
+
+void* __wrap__malloc_r(struct _reent* reent, size_t size) {
+  (void) reent;
+  return malloc(size);
+}
+
 void free(void* ptr) {
   vPortFree(ptr);
 }
 
+void __wrap_free(void* ptr) {
+  free(ptr);
+}
+
 void* calloc(size_t num, size_t size) {
-  (void)(num);
-  (void)(size);
+  (void) num;
```

Comment:
A random thought: any reason we don't implement this? Isn't it just `malloc(num * size); memset(ptr, 0, num*size)`? Perhaps I'm missing some alignment subtleties or size overflow or something

Reply:
No reason, I just left as it was. I'll go ahead and implement it, I guess the only concern would be overflow as you mentioned but I don't think it's a big issue.